.gitlab-ci.yml

```diff
@@ -9,7 +9,8 @@ variables:
     CARGO_TARGET: x86_64-unknown-linux-gnu
 
 .setup_rust: &setup_rust
-    rustup default nightly
+    rustup default nightly &&
+    rustup component add clippy
```

Comment:
this is not exactly where we want to have this.. clippy is not necessary for the release build job. this should be in `before_script` in test_cargo

Reply:
Fixed